onto-viewer-core/src/main/java/org/edmcouncil/spec/ontoviewer/core/comparator/ComparatorWithAlphabeticalOrder.java

```diff
@@ -27,6 +28,10 @@ public static Comparator<Object> get() {
         txt1 = ((OwlAxiomPropertyValue)obj1).getFullRenderedString().toLowerCase();
         txt2 = ((OwlAxiomPropertyValue)obj2).getFullRenderedString().toLowerCase();
       }
+      if (obj1.getClass().getName().equals(obj2.getClass().getName().equals("OwlListElementIndividualProperty"))) {
```

Comment:
This is always `false`, because it compares a `String` with a `boolean`. Fix this somehow and also you can fix the comparison that is used above.